airbyte-cdk/java/airbyte-cdk/db-destinations/src/testFixtures/java/io/airbyte/cdk/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -789,7 +789,7 @@ public void testIncrementalDedupeSync() throws Exception {
         .map(record -> Jsons.deserialize(record, AirbyteMessage.class))
         .collect(Collectors.toList());
     final JsonNode config = getConfig();
-    runSyncAndVerifyStateOutput(config, firstSyncMessages, configuredCatalog, true);
+    runSyncAndVerifyStateOutput(config, firstSyncMessages, configuredCatalog, false);
```

Comment:
since we're keeping normalization alive in destination-mysql, should we keep DATs running normalization?

iirc there's some way for individual destinations to disable normalization in DATs